src/components/Login.js

```diff
@@ -35,12 +60,18 @@ class Login extends Component {
                 headers: {
                     'Cache-Control': 'no-cache',
                     'Content-Type': 'application/json',
-                    "Access-Control-Allow-Origin": "*"
+
                 }
             })
             .then(resp => {
-                console.log(resp);
-                localStorage.setItem('tokenData', resp.data.tokenData);
```

Comment:
@AdamSarach, I know it isn't important right now but please consider to store token and refresh token in different places. Maybe we should discuss it during next meeting